TypeCobol.Test/Parser/Programs/Cobol85/ProgramEndMismatchMain.rdz-Nodes.txt

```diff
@@ -0,0 +1,11 @@
+Line 14[8,24] <37, Warning, General> - Warning: Program name not matching "Main".
```

Comment:
The IBM compiler error is:
`Program-name"TOTO"did not match the name of any open program.  The"END PROGRAM"marker was assumed to have ended program"MAIN".`
Can you change to match IBM message ?

Reply:
Messages should be similar now.